src/components/patterns/link.tsx

```diff
@@ -44,10 +44,14 @@ export const DrupalActionLink = ({href, children, ...props}: PropsWithChildren<{
     <Link
       href={href}
       {...props}
-      className={twMerge("relative pr-30 hocus:text-brick no-underline rs-mt-neg1 pt-10", props.className)}
+      className={'pt-10'}
```

Comment:
Let's go with something like this for simple strings:
```suggestion
      className="pt-10"
```